packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -59,28 +57,36 @@ const styles = StyleSheet.create({
   }
 })
 
-type AudioProps = ReturnType<typeof mapStateToProps> &
-  ReturnType<typeof mapDispatchToProps>
-
-const Audio = ({
-  track,
-  index,
-  queueLength,
-  playing,
-  seek,
-  play,
-  pause,
-  next,
-  previous,
-  reset,
-  repeatMode,
-  isShuffleOn,
-  shuffleIndex
-}: AudioProps) => {
-  const trackOwner = useSelector((state) =>
-    getUser(state, { id: track?.owner_id })
+export const Audio = () => {
+  const track = useSelector(getCurrentTrack)
+  const index = useSelector(getIndex)
+  const queueLength = useSelector(getLength)
+  const playing = useSelector(getPlaying)
+  const seek = useSelector(getSeek)
+  const repeatMode = useSelector(getRepeat)
+  const isShuffleOn = useSelector(getShuffle)
+  const shuffleIndex = useSelector(getShuffleIndex)
+  const trackOwner = useSelector(
```

Comment:
Yeah a bit confusing there. And so useproxyselector also doesn't do anything? What does this mean... that the users cache is updating when going to a user that has reposted the track your playing? Or going to the user of the playing track?

Reply:
Yeah the users cache is updating when going to the artist who posted the track. We are using it for the gateway and the name that displays on the music control. But let me try selecting those things specifically to avoid this

Reply:
going to merge this in for now, and then we can figure out why profile's gateways are changing

Reply:
Cool, it has to do with the collectibles fetch. We are getting diffent metadata back from that call and updating the user in cache. Might have it figured out by Monday!